Libplanet/Blockchain/BlockChain.cs

```diff
@@ -748,6 +761,13 @@ bool renderActions
                 // the tx nounce order when the block was created
                 foreach (Transaction<T> tx1 in block.Transactions)
                 {
+                    if (!Policy.IsValidTransaction(tx1))
+                    {
+                        throw new InvalidTxException(
```

Comment:
It may be better if we add some specific exception types instead of using concrete `InvalidTxException.`

Reply:
Added `InvalidTxByBlockPolicyException`.